vsts/vsts.yaml

```diff
@@ -17,210 +66,84 @@ resources:
     clean: true
 
 jobs:
-  ### Linux build 1 ###
-  - job: LINUX1
```

Comment:
All OS test targets collapsed into one job, rather that duplicated (DRY violation) into 2.